templates/webhook.bin.erb

```diff
@@ -173,6 +179,24 @@ end
 
   helpers do
 
+    # Ignore environments that we don't care about e.g. feature or bugfix branches
+    def ignore_env?(env)
+      list = $config['ignore_environments']
+      return false if list.nil? or list.empty?
+
+      list.each do |l|
+        # Even unquoted array elements wrapped by slashes becomes strings after YAML parsing
+        # So we need to convert it into Regexp manually
+        if l =~ /\/.+\//
```

Comment:
shouldn't this have start/end anchors to prevent matching things like `foo/bar/`?